src/gsi/obsmod.F90

```diff
@@ -496,7 +496,7 @@ module obsmod
 
   ! 1/237: Dropsonde observations.
 
-  ! 292: SFMR observations.
+  ! 213: SFMR observations.
```

Comment:
Similar comment as gsimod.F90.   Shouldn't uv_doe_a_292 and uv_doe_b_292 be uv_doe_a_213 and uv_doe_a_213?

Reply:
Yes, I thought about this, since uv_doe_a_292 etc are variables and there are quite a few of this, they are appeared as well as in the HWRF namelist and HWRF conf file. If change this to uv_doe_a_213 etc. the changes are quite a lot.

Reply:
"Changes are a lot" - would you please explain.   I see changes to three source code files:  gismod.F90, obsmod.F90, and setupspd.f90.   Where do the HWRF namelist and conf reside?  Are they in lbi2018:release/gfsda.v16.0.0.HDOB or the NOAA-EMC/GSI master?

Reply:
Sorry for not being clear. No, the HWRF namelist conf are not part of  lbi2018:release/gfsda.v16.0.0.HDOB. They are part of the HWRF workflow. Let me talk to the HWRF team, since we have finalized the HWRF v16 workflow.

Reply:
updated